### PR TITLE
NIAD-5: Enhance the documentation of monitoring and audit

### PR DESCRIPTION
**Pull request by Broom94**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -142,3 +142,4 @@ mhs-reference-implementation/selenium_tests/data/local_asid
 
 # system
 .DS_Store
+integration-tests/integration_tests/integration-tests-env.yaml
```
Comment (mattd-kainos) [inline .gitignore]: already ignored under #Local configuration section?

File: mhs/operating-mhs-adaptor.md [hunk 2]
```diff
@@ -3,19 +3,27 @@
...
 ## Log consumption
-The MHS services emit logs and audit records on standard I/O streams, following [The Twelve Factor App](http://12factor.net) guidance on [logging](https://www.12factor.net/logs). Logs within the MHS Adaptor context are therefore captured 
-by the Docker containers they are hosted within. Whichever Docker container orchestration technology is used, these log streams can be captured and/or 
-forwarded to an appropriate log indexing service for consumption, storage and subsequent query.
+The MHS services emit logs and audit records on standard I/O streams which are captured by the Docker containers they are hosted within. 
+Whichever Docker container orchestration technology is used, these log streams can be captured and/or forwarded to an appropriate log indexing 
+service for consumption, storage and subsequent query.
 
-The [AWS exemplar](README.md#aws-exemplar) deployment of the MHS Adaptor makes use of the ECS Fargate service and the logs are captured into the 
+The [AWS exemplar](README.md#aws-exemplar) deployment of the MHS makes use of the ECS Fargate service and the logs are captured into the
```
Comment (mattd-kainos) [inline mhs/operating-mhs-adaptor.md]: MHS Adaptor is probably the correct language here

File: mhs/operating-mhs-adaptor.md [hunk 3]
```diff
@@ -91,30 +59,215 @@ An example is given below:
...
+| 17 | `Retrieving latest work description to check version` | Queries state database with local state to check remote state exists |
+| 18 | `Retrieved previous version, comparing versions` | Previous remote version found in state database and comparing versions |
+| 19 | `Local version matches remote, incrementing local version number` | Checks versions to avoid collisions. Previous remote version matches local version, increments local version number | 
+| 20 | `Successfully updated work description to state store for key=C78164D9-A1EB-497E-BCB5-74CABD3162F0` | Successfully updated state database with message status updated, spine successfully received message. | 
+
+### Cloud Watch
+
+See below example to write CloudWatch queries to parse logs and search for a given correlation id and filter out healthchecks. 
+
+ `parse '* | * | * | * | * | * | * | * | *' as timestamp, level, correlation_id, message_id, interaction_id, inbound_message_id, process, name, text `<br>
```
Comment (mattd-kainos) [inline mhs/operating-mhs-adaptor.md:262]: Rested this query after https://github.com/nhsconnect/integration-adaptors/pull/200/files was merged and it correctly captures outbound logs. I didn't see any for inbound though.

**Review by mattd-kainos — APPROVED** [review]

**Comment by jaff47** [comment]
Approved

**Pull request merged**